app/renderer/js/utils/system-util.ts

```diff
@@ -27,29 +31,30 @@ class SystemUtil {
 		return instance;
 	}
 
-	getOS() {
-		if (os.platform() === 'darwin') {
+	getOS(): string {
+		const platform = os.platform();
+		if (platform === 'darwin') {
 			return 'Mac';
-		}
-		if (os.platform() === 'linux') {
+		} else if (platform === 'linux') {
 			return 'Linux';
-		}
-		if (os.platform() === 'win32' || os.platform() === 'win64') {
+		} else if (platform === 'win32') {
 			if (parseFloat(os.release()) < 6.2) {
 				return 'Windows 7';
 			} else {
 				return 'Windows 10';
 			}
+		} else {
+			return '';
 		}
 	}
 
-	setUserAgent(webViewUserAgent) {
-		this.userAgent = 'ZulipElectron/' + app.getVersion() + ' ' + webViewUserAgent;
+	setUserAgent(webViewUserAgent: string): void {
+		this.userAgent = `ZulipElectron/'${app.getVersion()}${webViewUserAgent}`;
```

Comment:
@priyank-p, @vsvipul can you guys make sure this is correct?

Reply:
I think this should be 
> `ZulipElectron/${app.getVersion()} ${webViewUserAgent}`

Reply:
@priyank-p Can you fix this please ?